cluster/gce/util.sh

```diff
@@ -1786,7 +1837,7 @@ function test-setup() {
   # Detect the project into $PROJECT if it isn't set
   detect-project
 
-  if [[ ${MULTIZONE:-} == "true" ]]; then
+  if [[ ${MULTIZONE:-} == "true" && -n ${E2E_ZONES:-} ]]; then
```

Comment:
We need to distinguish two cases: 
1. we want to set-up multimaster cluster with kubelets in a single zone (MULTIZONE=true, E2E_ZONES=false) a
2. we want to set-up cluster with kubelets in many zones (MULTIZONE=ture, E2E_ZONES={list of zones for kubelets})